test/AutobahnTestServer/project.json

```diff
@@ -1,12 +1,14 @@
 {
-  "exclude": "wwwroot/**/*",
   "dependencies": {
     "Microsoft.NETCore.Platforms": "1.0.1-*",
     "Microsoft.AspNetCore.WebSockets.Server": "0.1.0-*",
     "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*"
   },
   "buildOptions": {
-    "emitEntryPoint": true
+    "emitEntryPoint": true,
+    "compile": {
+      "exclude": "wwwroot/**/*"
```

Comment:
Agreed. Also remove the redundant `hosting.json` later on in the file. That no longer exists.